### PR TITLE
[ADDED] rs about doc

### PR DESCRIPTION
**Pull request by oldbilldue18**
Completed the assigned issue #142

### CODE REVIEW

**Review by aryadubey03 — CHANGES_REQUESTED** [review]
there are several spelling mistakes. Please check properly and do changes.

File: docs/projects/resume-screener-website/about-resume-screener-website.mdx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+>_This is the official documentation of **[Resume Screener Website](https://rs.tcetmumbai.in/)** of **[TCET Mumbai](https://tcetmumbai.in)**, maintained by the **[TCET Open Source](https://opensource.tcetmumbai.in)** Documentation team._
+
+import pic from "/docs/projects/resume-screener-website/assets/about.jpg";
+
+<p align="center">
+<img src={pic} style={{ border: "2px solid gray" }} /> 
+</p>
+
+<br /><br />
```
Comment (aryadubey03) [inline docs/projects/resume-screener-website/about-resume-screener-website.mdx:16]: Too much space here

**Review by aryadubey03 — CHANGES_REQUESTED** [review]

**Review by aryadubey03 — CHANGES_REQUESTED** [review]
Screenshot (208)

You have left too much space here.

**Review by aryadubey03 — APPROVED** [review]

**Review by himanshu-03 — APPROVED** [review]
LGTM!

**Pull request merged**